MTA-6893 Document Hub OIDC authentication (#382)
migtools/mta-documentation: docs/topics/mta-cli/proc_connect-to-hub.adoc

:_newdoc-version: 2.18.3
:_template-generated: 2025-03-17
:_mod-docs-content-type: PROCEDURE

[id="connect-cli-hub_{context}"]
= Connecting the CLI to the {ProductShortName} hub

[role="_abstract"]

> To use organization standard configuration for analysis and rules, you must authenticate the command-line (CLI) to the {ProductFirstRef} hub. 

Reviewer: ```suggestion
To use organization-standard configurations for analysis and rules, authenticate the command-line interface (CLI) to the {ProductFirstRef} Hub.
```

.Prerequisites

> * You installed {ProductShortName} 8.2.0 in a Red{nbsp}Hat {ocp-name} cluster.

Reviewer: ```suggestion
* You have installed {ProductShortName} 8.2.0 on a Red{nbsp}Hat {ocp-name} cluster.
```

.Procedure

. Log in to the {ProductShortName} Hub:
+
[subs="+quotes"]
----
$ mta-cli config login
----

. Enter the `Host` to connect to the Hub:
+
[subs="+quotes"]
----
Host: \https://mta-namespace.apps.cluster.example.com/hub
----

> * The `Host` is the {ProductShortName} web console URL that exposes the `hub` endpoint.

Reviewer: ```suggestion
+
The `Host` value is the {ProductShortName} web console URL that exposes the `hub` endpoint.
```

. Open the displayed OpenID Connect (OIDC) device authentication URL in your browser.

. Log in by using the {ProductShortName} web console credentials.

. Copy the user code displayed in the {ProductShortName} {CLIName}.

> . Enter the user code in the browser link to authenticate the device.

Reviewer: ```suggestion
. Enter the user code in the browser to authenticate the device.
```

"Browser link" is awkward phrasing. Users do not enter text into a link; they enter text into a field on a web page. Simplifying this to "in the browser" makes the instruction clearer.
